app/src/AppRouter.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Header from './components/organisms/Header';

```

Comment:
For all the imports in this file, remove the ./ at the beginning for absolute imports